designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ExpressionSuggesterSpec.scala

```diff
@@ -64,7 +72,11 @@ class Util {
   def withDescription(): Int = 42
 }
 
-class ExpressionSuggesterSpec extends AnyFunSuite with Matchers with PatientScalaFutures {
```

Comment:
I just realize that the `ExpressionSuggester` and it tests are in `designer` instead of `interpreter`. It is insane because the main tested logic (`SpelExpressionSuggester`) is in `interpreter`. Can you move it to `interpreter`?

Reply:
That would require modyfing the tests - the ExpressionSuggester and CaretPosition2d under test are from designer module.

Reply:
Eh, so let it be as it is.